package.json

```diff
@@ -24,7 +24,7 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "readable-stream": "~2.0.5"
+    "readable-stream": "~2.2.2"
```

Comment:
I would love to, but I wasn't sure if that was the style `bl` used.  Want me to switch it?